src/main/resources/gtceu.mixins.json

```diff
@@ -1,81 +1,82 @@
 {
-    "required": true,
-    "minVersion": "0.8",
-    "refmap": "gtceu.refmap.json",
-    "package": "com.gregtechceu.gtceu.core.mixins",
-    "compatibilityLevel": "JAVA_17",
-    "plugin": "com.gregtechceu.gtceu.core.mixins.GTMixinPlugin",
-    "client": [
-        "client.AbstractClientPlayerAccessor",
-        "client.BakedQuadMixin",
-        "client.BiomeColorsMixin",
-        "client.BlockModelMixin",
-        "client.ClientLevelAccessor",
-        "client.FaceBakeryMixin",
-        "client.GuiGraphicsAccessor",
-        "client.GuiGraphicsMixin",
-        "client.GuiHeartTypeMixin",
-        "client.HumanoidArmorLayerMixin",
-        "client.LevelRendererMixin",
-        "client.ModelManagerMixin",
-        "client.MultiPlayerGameModeMixin",
-        "client.PlayerInfoAccessor",
-        "client.VariantDeserializerMixin",
-        "ftbchunks.FTBChunksClientMixin",
-        "ftbchunks.LargeMapScreenMixin",
-        "ftbchunks.RegionMapPanelMixin",
-        "ldlib.ModularWrapperWidgetMixin",
-        "rei.FluidEntryRendererMixin",
-        "xaerominimap.HighlighterRegistryMixin",
-        "xaerominimap.MinimapFBORendererMixin",
-        "xaeroworldmap.GuiMapMixin",
-        "xaeroworldmap.MapElementRenderHandlerBuilderMixin",
-        "xaeroworldmap.WorldMapSessionMixin"
-    ],
-    "mixins": [
-        "AnvilMenuMixin",
-        "BlockBehaviourAccessor",
-        "BlockMixin",
-        "BlockPropertiesAccessor",
-        "ChunkGeneratorMixin",
-        "EntityMixin",
-        "IngredientAccessor",
-        "InventoryMixin",
-        "ItemValueAccessor",
-        "LevelMixin",
-        "LootDataManagerMixin",
-        "LootPoolAccessor",
-        "OreConfigurationMixin",
-        "OreVeinifierMixin",
-        "PotionBrewingAccessor",
-        "PrimedTntAccessor",
-        "RecipeManagerMixin",
-        "RepairItemRecipeMixin",
-        "ServerChunkProviderMixin",
-        "ServerGamePacketListenerImplAccessor",
-        "ShapedRecipeAccessor",
-        "SidedRedstoneConnectivityMixin",
-        "TagLoaderMixin",
-        "TagManagerMixin",
-        "TagValueAccessor",
-        "emi.EmiRecipeFillerMixin",
-        "emi.FillRecipePacketMixin",
-        "emi.FluidEmiStackMixin",
-        "forge.ConfiguredModelBuilderAccessor",
-        "forge.ConfiguredModelListAccessor",
-        "forge.IntersectionIngredientAccessor",
-        "forge.PartialNBTIngredientAccessor",
-        "forge.StrictNBTIngredientAccessor",
-        "jei.FluidHelperMixin",
-        "ldlib.SyncUtilsMixin",
-        "registrate.AbstractRegistrateAccessor",
-        "registrate.RegistrateDataProviderAccessor",
-        "rei.InputSlotCrafterMixin",
-        "rei.RecipeFinderMixin",
-        "top.ConfigMixin"
-    ],
-    "injectors": {
-        "defaultRequire": 1,
-        "maxShiftBy": 5
-    }
+  "required": true,
```

Comment:
Can you restore the indentation as it was? That way it won't clash with PRs that were made before/after